modules/base/tool.go

```diff
@@ -45,8 +45,8 @@ func EncodeSha1(str string) string {
 }
 
 func ShortSha(sha1 string) string {
-	if len(sha1) == 40 {
```

Comment:
I have done this to always have uniform short tag on all pages regardless of what users use (7-char, 8-char, ..., 40-char commit  SHA).

Ok if you want remove this; I will do needed modifications tonight.